keyboards/mode/eighty/keymaps/rules.mk

```diff
@@ -0,0 +1,2 @@
+VIA_ENABLE = yes
```

Comment:
This file doesn't belong here.  You need to move it to the VIA keymap folder, or delete it.